lxd/devices.go

```diff
@@ -358,8 +457,63 @@ func deviceNetworkPriority(d *Daemon, netif string) {
 	return
 }
 
+func deviceUSBEvent(d *Daemon, usb usbDevice) {
+	containers, err := dbContainersList(d.db, cTypeRegular)
+	if err != nil {
+		shared.Log.Error("problem loading containers list", log.Ctx{"err": err})
+		return
+	}
+	for _, name := range containers {
+		containerIf, err := containerLoadByName(d, name)
+		if err != nil {
+			continue
+		}
+
+		c, ok := containerIf.(*containerLXC)
```

Comment:
Not fond of this I must say but don't have a good alternative to suggest right now. If we have to do more of that in the future, we'll have to think about how we can better handle such cases within the Container interface.